bus-mapping/src/circuit_input_builder/tracer_tests.rs

```diff
@@ -482,6 +482,142 @@ fn tracer_err_address_collision() {
     );
 }
 
+#[test]
+fn tracer_create_without_collision() {
```

Comment:
`tracer_create_collision_free` more precise

Reply:
updated in [c5dcce4](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1326/commits/c5dcce4fb6e4c8659c6e56696603925d4203c908) thanks!